python_modules/dagster/dagster/_core/definitions/sensor_definition.py

```diff
@@ -402,6 +409,11 @@ def repository_def(self) -> Optional["RepositoryDefinition"]:
         """Optional[RepositoryDefinition]: The RepositoryDefinition that this sensor resides in."""
         return self._repository_def
 
+    @property
+    def code_location_origin(self) -> Optional["CodeLocationOrigin"]:
```

Comment:
I think i'd prefer to keep this a non-public property for now. Otherwise, users may start to use it in the body of their sensor functions and then will need a way to make `CodeLocationOrigin`s and pass them to `build_sensor_context` for testing their sensors